app/Models/EventUserRegistered.php

```diff
@@ -40,11 +40,11 @@ class EventUserRegistered extends Model
 
     protected $guarded = [];
 
-    public function event(){
+    public function event() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->